packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -60,10 +69,15 @@ function UncontrolledInnerBlocks( props ) {
 		orientation,
 		placeholder,
 		layout,
+		name,
+		blockType,
+		innerBlocks,
+		parentLock,
```

Comment:
Isn't that a public component? so we're adding props to a public component?

Reply:
No, only `useInnerBlocksProps` is public and a component wrapper around that.

Reply:
(this component is not even exported)